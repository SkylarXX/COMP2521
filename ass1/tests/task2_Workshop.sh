./task2 tests/Workshop
