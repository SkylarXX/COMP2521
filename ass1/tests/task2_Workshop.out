
=== Tree ===
            <Pine.LNX.4.64.1111301118230.14763@cse.unsw.EDU.AU>
         <Pine.LNX.4.64.1111291922080.7025@kettledrum.orchestra.cse.unsw.EDU.AU>
            <FF1D31B8-4C4D-4742-B785-60153400289F@cse.unsw.edu.au>
                     <FE4653F7-DAAE-49FF-9A31-0A89E1AF5044@cse.unsw.edu.au>
                  <FAD89FCA-B20F-47C9-9036-46FC0E40CA42@cse.unsw.edu.au>
               <F43451FB-6F23-491D-8BD7-A6285AFB67EC@cse.unsw.edu.au>
                     <F3BB7E77-37BA-49CA-8853-C98CEEEA4050@cse.unsw.edu.au>
                  <EA5F7A26-5ADF-4DF3-8203-8991D9E4282A@cse.unsw.edu.au>
      <E8B62DAC-9F40-49E2-BA9F-6DFA70A8D322@cse.unsw.edu.au>
         <E05D668D-DE51-4166-9C9B-891688ADB210@cse.unsw.edu.au>
               <DD2B3A55-13F6-40BC-8615-5F14D4E542AC@cse.unsw.edu.au>
                  <CALQ3V4KqG9wL6tX7bAPU5Z6RBCDi7DpA4+PF_KgerNGqg+_HfQ@mail.gmail.com>
                     <CALQ3V4KeRfKrm1jP0EtPM5+vNRPe8JtY2jUUfM=EPRQ2Up7LEg@mail.gmail.com>
                        <CALQ3V4KJ30ith8h68+d3P_tLXf2GMax5PkxKf8NGO=M9nwp-3w@mail.gmail.com>
            <CAD38MSFRVsMtX35P10qcPqs1k1NcYYJVZ5yz2hcfChY4Y1LpHg@mail.gmail.com>
   <CA+uZbSofVTVfqf3ULohiPvx8R2Kwn=urZpqxeZLxtbZUR55UZw@mail.gmail.com>
                     <C7A1685E-2A53-4400-AD25-2DA358A6987F@cse.unsw.edu.au>
                                 <C6CEFFFC-6C43-4FB2-81B7-355850AF0BB2@cse.unsw.edu.au>
                                    <C3A06CDD-4D47-419D-B6E9-1308BE28E11D@cse.unsw.edu.au>
                                       <B602879E-CD3F-4FCD-B3C1-51243092209C@cse.unsw.edu.au>
                              <B049B2B1-D4C0-4CD2-A0AD-43513A9D920B@cse.unsw.edu.au>
                                    <AE66195C-639B-4E4B-931E-7C139A220CE8@cse.unsw.edu.au>
                                 <ACDEF6F6-257F-4A6F-8755-2BC9E8BBE134@cse.unsw.edu.au>
                           <A8BACAB4-084D-424B-8D3C-E6AE65658A9F@cse.unsw.edu.au>
                              <A844DEED-2655-4E09-8E79-BA53E95DB70C@cse.unsw.edu.au>
                        <99EAAD38-7A42-4C31-98A9-4267E4794E20@cse.unsw.edu.au>
                           <95367193-11B5-48E3-B929-421A7BC7F138@cse.unsw.edu.au>
                  <8803EC9B-6EAE-422D-972D-4F02B236F2DB@cse.unsw.edu.au>
               <6AB2D6D8-BF45-40F7-BF4A-ACA273179839@cse.unsw.edu.au>
            <65FA592B-80D1-4A89-AA71-BE199AC470C1@cse.unsw.edu.au>
                     <5FABA40C-0352-4DF4-8AC8-155DEEC94887@cse.unsw.edu.au>
                  <57428E5F-CC9D-444D-8FAB-CC88249054EA@cse.unsw.edu.au>
               <4ED31A6C.3000504@cse.unsw.edu.au>
         <4ED2E3A5.6080604@cse.unsw.edu.au>
      <4ED2DF07.4010207@cse.unsw.edu.au>
<40FE8F93-83A7-4EB4-87D2-C3CCE42AAAE9@cse.unsw.edu.au>
      <3D0D4358-0F86-4BC5-B7EB-F806606A5D5A@cse.unsw.edu.au>
               <37F601BA-FD32-493E-BDC7-5ABE148C6D69@cse.unsw.edu.au>
            <2F6BD270-B773-4EC9-8E14-0AF23E2C2920@cse.unsw.EDU.AU>
                        <2F3FC28A-3699-4152-8D12-98B458010EC2@cse.unsw.edu.au>
                                 <299F345B-B944-4C87-884C-52E2F493850F@cse.unsw.edu.au>
                              <290CCAA6-0962-471A-A57D-807AE51C4CC7@cse.unsw.edu.au>
                                 <28052D39-2D48-4A92-B521-65C8BEBD7EB5@cse.unsw.edu.au>
                           <21EE5DA5-CCC7-4D78-944B-EAD35B0D8A73@cse.unsw.edu.au>
                     <20111129211623.GX14327@cse.unsw.edu.au>
                  <20111129175346.GA6258@cse.unsw.EDU.AU>
               <20111129144226.GB28560@cse.unsw.edu.au>
         <20111129090123.GA7125@cse.unsw.EDU.AU>
               <1C13A30E-64F8-4B70-A2C5-E4D56DAEC699@cse.unsw.edu.au>
            <1BFCE748-18FA-42DA-954F-86BEBF335FAA@cse.unsw.edu.au>
               <1BD4F884-0713-41C9-AC3F-367EEB8CC0A7@cse.unsw.edu.au>
                  <130F095F-8328-4AE7-A04E-400BCC65ECBF@cse.unsw.edu.au>
   <1204DCB5-E43D-4AAD-88B9-C5F95016E505@me.com>
